readme

```diff
@@ -0,0 +1 @@
+this is just a file to make a pull request
```

Comment:
test

Reply:
Although some of the classes where confusing like app.html line 80 and 81. I liked the site on the whole. the links all worked well. the comments on the html page where descriptive and well formatted for the most part, although there were a few lines with extra spaces and lines. looking  at the variables and classes, it seems that they were used well in the program, with only the button classes and types being a bit confusing for me. Also the internal styling used not too sure that would be necessary as at present is seems to be causing some issues on the page. Looking at the searchGoogle.js i liked that you tried to use asyc functions although there's a bit too many comments present. I think with some more time you guys could have got the search to work but so far it looks good.